volume-providers/vpc/util/unit_util_test.go

```diff
@@ -0,0 +1,493 @@
+/*******************************************************************************
+ * IBM Confidential
+ * OCO Source Materials
+ * IBM Cloud Container Service, 5737-D43
+ * (C) Copyright IBM Corp. 2018 All Rights Reserved.
+ * The source code for this program is not  published or otherwise divested of
+ * its trade secrets, irrespective of what has been deposited with
+ * the U.S. Copyright Office.
+ ******************************************************************************/
+
+package util
```

Comment:
Please update this file as per actual functions in the imp file

Reply:
DONE